tools/stats/print_test_stats.py

```diff
@@ -794,7 +794,7 @@ def process_intentional_test_runs(runs: List[TestCase]) -> Tuple[int, int]:
 
 def assemble_flaky_test_stats(duplicated_tests_by_file: Dict[str, DuplicatedDict]) -> Any:
     flaky_tests = []
-    workflow_id = os.environ.get("WORKFLOW_ID", os.environ.get("CIRCLE_WORKFLOW_ID", None))
+    workflow_id = os.environ.get("GITHUB_RUN_ID", os.environ.get("CIRCLE_WORKFLOW_ID", None))
```

Comment:
`GITHUB_RUN_ID`  I think corresponds to a workflow, we probably want `GITHUB_JOB` instead